src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -35,6 +35,8 @@ namespace Microsoft.AspNet.Mvc.TagHelpers
     public class LinkTagHelper : UrlResolutionTagHelper
     {
         private static readonly string Namespace = typeof(LinkTagHelper).Namespace;
+        private static readonly string FallbackJavaScriptResourceName = 
+            Namespace + ".compiler.resources.LinkTagHelper_FallbackJavaScript.js";
```

Comment:
Could you just inline `Namespace`. It seems like such a poor variable \ field name.